src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3480,10 +3488,15 @@ internal void UpdateTypes(ExecutionContext context, bool updateOnly)
             // Use at most 3 locks (we don't expect contention on that many cores anyways,
             // and typically we'll be processing just 2 or 3 files anyway, hence capacity=3.
             ConcurrentDictionary<string, string> filesProcessed
-                = new ConcurrentDictionary<string, string>(/*concurrencyLevel*/3, /*capacity*/3, StringComparer.OrdinalIgnoreCase);
-            Parallel.ForEach(Types, sste =>
-            //            foreach (var sste in Types)
-            {
+                = new ConcurrentDictionary<string, string>(
+                    concurrencyLevel: 3,
+                    capacity: 3,
+                    StringComparer.OrdinalIgnoreCase);
```

Comment:
The line break before `=` doesn't seem necessary. The rest is good though! 👍

Reply:
I will fix in last commit or in follow PR - the file contains still many style issues.

Reply:
Appreciate you taking the time to clean up old code in such an organised manner! 💖 😊

Reply:
We have already seen that it significantly reduces the time for PR passing through code review.